downstream/modules/eda/con-event-streams.adoc

```diff
@@ -0,0 +1,39 @@
+
+[id="event-streams"]
+
+= Event streams
+
+[role="_abstract"]
+Event streams can send events from remote systems to {EDAcontroller}. In a typical set-up, a server sends data to an event stream over the internet to an {EDAName} event stream receiver. When the data comes over the internet, the request must be authenticated. Depending on the webhook vendor or remote system, the authentication method could differ.
+
+{EDAcontroller} supports 7 different event stream types.
```

Comment:
seven

Reply:
I checked both the Red Hat Supplementary guide and the IBM Style Guide. I tend to lean towards the latter. Earlier in my career, that rule was spell out all numbers less than 10, but later AP style guide and others were less restrictive. Currently, here's what IBM says [here](https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words): 

"You can choose whether to use numerals or words for numbers that are less than 10.

Whichever approach you choose, use it consistently within a single publication."

See screen capture also
![Screenshot from 2024-09-27 10-42-18](https://github.com/user-attachments/assets/02b22086-2d00-4f46-a7f0-023d3422590e)